picodom.d.ts

```diff
@@ -1,26 +1,41 @@
-export interface VNode<Data> {
+export as namespace picodom
+
+export interface VirtualNode<Data> {
   tag: string
-  data?: Data
+  data: Data
```

Comment:
Per [line 26 in h.js](https://github.com/picodom/picodom/blob/901a0c8a403244cfc8ec0d143aca036b208ba491/src/h.js#L26), data will never be undefined.